autoload/maktaba/syscall.vim

```diff
@@ -1,5 +1,8 @@
 "" Utilities for making system calls and dealing with the shell.
 
+let s:callbacks = {}
```

Comment:
These initializations need to be guarded with `if exists(…)` or they'll be blown away by `:call maktaba#syscall#NonexistentFunc()` or anything that triggers vim to re-source.